src/tests/extensions/authenticationHelper_tests.ts

```diff
@@ -1,13 +1,18 @@
 import * as sinon from "sinon";
 
-import {AuthenticationHelper} from "../../scripts/extensions/authenticationHelper";
-import {ClipperData} from "../../scripts/storage/clipperData";
-import {Logger} from "../../scripts/logging/logger";
-import {UserInfoData} from "../../scripts/userInfo";
+import { AuthenticationHelper } from "../../scripts/extensions/authenticationHelper";
```

Comment:
I'll remove them.  VSCode started doing it automatically for some reason.  I'll try to turn it off.